src/components/tools/polygon/PolygonTool.vue

```diff
@@ -181,13 +189,27 @@ export default defineComponent({
 
     const currentTools = useCurrentTools(activeToolStore, viewAxis);
 
+    const { onHover, overlayInfo } = useHover<ToolID>(
+      currentTools,
+      currentSlice
+    );
+
+    const pointsToBound = computed(() => {
```

Comment:
Minor: this name confused me, since it's just the visible tool points and not a bounding box.